src/main/webapp/teacher/controlGameInstance/controlGameInstance.js

```diff
@@ -299,8 +351,21 @@ function updateCurrentQuestionStats(updatedQuestionStats) {
   const numberWrongCurrentQuestionAnswersElement = document.getElementById('jsNumberWrongCurrentQuestionAnswers');
   numberWrongCurrentQuestionAnswersElement.innerText = 'Number of answers wrong: ' + updatedQuestionStats.numberWrong
 
-  // Update the question in the history panel
+}
+
+function updateCurrentQuestionStatsInHistory(updatedQuestionStats, currentQuestionId) {
+  const numberQuestionAnswersHistoryElement = document.getElementById('stats-' + currentQuestionId + '-numberQuestionAnswers');
+  if (numberQuestionAnswersHistoryElement == null) {
+    // This may be executed before the node is created the first time
+    return;
+  }
+  numberQuestionAnswersHistoryElement.innerText = 'Number of answers: ' + updatedQuestionStats.numberAnswered;
+
+  const numberCorrectQuestionAnswersHistoryElement = document.getElementById('stats-' + currentQuestionId + '-numberCorrectQuestionAnswers');
+  numberCorrectQuestionAnswersHistoryElement.innerText = 'Number of correct anwers: ' + updatedQuestionStats.numberCorrect;
 
+  const numberWrongQuestionAnswersHistoryElement = document.getElementById('stats-' + currentQuestionId + '-numberWrongQuestionAnswers');
```

Comment:
NIT: If you are creating the element here (line 117) mayyyybe it's better if you pass the element as an argument instead of creating these complex ids?